app/components/TopNav.vue

```diff
@@ -243,5 +270,10 @@
   .theme-toggle__icon--sun {
     display: none;
   }
+
+  .badge--new {
+    background: #8736e0;
+    color: #ffffff;
+  }
```

Comment:
same as above

Reply:
oops - we have these colors already
just pushed an update